packages/redux/src/orders/__tests__/selectors.test.ts

```diff
@@ -132,19 +132,21 @@ describe('orders redux selectors', () => {
 
   describe('isTrackingsLoading()', () => {
     it('should get the trackings isLoading property from state', () => {
-      const spy = jest.spyOn(fromOrders, 'getTrackings');
+      const spy = jest.spyOn(fromOrders, 'getShipmentTrackings');
 
       expect(selectors.isTrackingsLoading(mockState)).toEqual(false);
```

Comment:
What do you think of renaming `isTrackingsLoading` selector to `areShipmentTrackingsLoading`?

Reply:
✅